subprojects/core-api/src/main/java/org/gradle/api/resources/TextResourceFactory.java

```diff
@@ -59,7 +59,7 @@ public interface TextResourceFactory {
      * Creates a text resource backed by the given file.
      *
      * @param file a text file evaluated as per {@link org.gradle.api.Project#files(Object...)}
-     * @param charset the file's character encoding (e.g. {@code "utf-8"})
+     * @param charset the file's character encoding (e.g. {@link java.nio.charset.StandardCharsets.UTF_8})
```

Comment:
somehow it seems not found:

despite adding the import like suggested for `sun.nio`:

<img width="879" height="532" alt="Image" src="https://github.com/user-attachments/assets/8c6df394-58f8-4485-b327-fcadeb84b2aa" />

<img width="943" height="748" alt="Image" src="https://github.com/user-attachments/assets/2e008c35-09de-46f9-8383-846376736049" />

<img width="970" height="777" alt="Image" src="https://github.com/user-attachments/assets/bd7269e1-00ac-459f-8d80-d6fdccd33d37" />